lib/std/build/InstallRawStep.zig

```diff
@@ -13,6 +13,7 @@ const fs = std.fs;
 const io = std.io;
 const sort = std.sort;
 const warn = std.debug.warn;
+const nl = std.cstr.line_sep;
```

Comment:
shouldn't the newline depend on the target, not the host system running the compiler?

Reply:
Upon further investigation it appears to always end in CRLF. Source: https://developer.arm.com/documentation/ka003292/latest and also running `objcopy -O ihex test.o test.hex` on a native .o file for a x86_64 linux system. So I'll update it to always to `\r\n`